HISTORY.md

```diff
@@ -1,3 +1,36 @@
+# TileDB v2.22.0 Release Notes
+
+## Deprecation announcements
+
+* Support for downloading dependencies with CMake `ExternalProject`s by specifying `-DTILEDB_VCPKG=OFF` will be removed in 2.23. Vcpkg will be downloaded and used to manage dependencies by default. See https://github.com/TileDB-Inc/TileDB/blob/dev/doc/dev/BUILD.md for help on how to disable automatically downloading vcpkg and acquire dependencies from the system.
+
+## Configuration changes
+
+* Add `vfs.gcs.service_account_credential` config option that specifies a Google Cloud service account credential JSON string. [#4855](https://github.com/TileDB-Inc/TileDB/pull/4855)
+* Add `vfs.gcs.external_account_credential` config option that specifies a Google Cloud Workload Identity Federation credential JSON string. [#4855](https://github.com/TileDB-Inc/TileDB/pull/4855)
```

Comment:
```suggestion
* Add `vfs.gcs.service_account_key` config option that specifies a Google Cloud service account credential JSON string. [#4855](https://github.com/TileDB-Inc/TileDB/pull/4855)
* Add `vfs.gcs.workload_identity_configuration` config option that specifies a Google Cloud Workload Identity Federation credential JSON string. [#4855](https://github.com/TileDB-Inc/TileDB/pull/4855)
```

The config options were subsequently renamed (I had forgotten to update the PR description). https://github.com/TileDB-Inc/TileDB/blob/a6bdf20e5065c64fdfe08225beb5a22d00f53ce4/tiledb/api/c_api/config/config_api_external.h#L394-L406